components/ApplicationGrantAmountSelector/ApplicationGrantAmountSelector.test.jsx

```diff
@@ -0,0 +1,121 @@
+import {
+  /*ApplicationGrantAmountSelector,*/ handleOnChange,
```

Comment:
Let's remove this if we are not using it.

```suggestion
  handleOnChange,
```